ui/src/views/Rules/ListRules/index.jsx

```diff
@@ -1285,12 +1370,33 @@ function ListRules(props) {
   }, [hashQuery, filteredRulesCount]);
 
   return (
-    <Dashboard title="Rules">
+    <Dashboard
+      title={
+        rewindDate ? `Rules @ ${rewindDate.toString().split('(')[0]}` : 'Rules'
+      }>
       {isLoading && <Spinner loading />}
       {error && <ErrorPanel fixed error={error} />}
       {!isLoading && productChannelOptions && (
         <Fragment>
           <div ref={searchFieldRef} className={classes.options}>
+            <DateTimePicker
+              disableFuture
```

Comment:
note: This doesn't seem to disable future times - only future dates. I imagine this is a bug in the underlying component we're using - so until we upgrade to something supported there's probably nothing we can do about it.